java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -647,7 +647,7 @@ void testQuantilesOnIntegerInput() {
       for (int j = 0 ; j < quantiles.length ; j++) {
         for (int i = 0 ; i < methods.length ; i++) {
           try(Scalar result = cv.quantile(methods[i], quantiles[j])) {
-            assertEquals(exactExpected[i][j], result.getDouble(), DELTA);
+            assertEqualsWithEpsilon(exactExpected[i][j], result.getDouble(), DELTA);
```

Comment:
So is this an Epsilon or a delta?  I am not sure that using greek letters is clear enough to denote what we want. I would almost rather see it be explicit like `assertEqualsWithinPercentage`.